js/menu.js

```diff
@@ -0,0 +1,18 @@
+const openMenu = document.getElementById('menu');
+const closeMenu = document.getElementById('close');
+const menu = document.getElementById('mob-nav');
+
+function openMobileMenu() {
+  menu.style.display = 'block';
+  openMenu.style.display = 'none';
+  closeMenu.style.display = 'block';
+}
+
+function closeMobileMenu() {
```

Comment:
- [ ] When I open the menu and close it, the hamburger button doesn't disappear even when I switch to the desktop version. Please double-check on that! 😄 
_See the marked image down below._
![Screenshot](https://user-images.githubusercontent.com/47625407/129447174-dd85f01d-23ee-48b5-8766-d53ca05d0156.png)